plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -189,6 +192,35 @@ public final void testBundleVendor() throws IOException {
     assertEquals("Google Inc.", vendor);
   }
 
+  @Test
+  public final void testGuavaImportVersions() throws IOException {
+    checkVersionStringsInManifestMf(
+        "Import-Package", "com.google.common.", "version=\"[20.0.0,21.0.0)\"");
+    checkVersionStringsInManifestMf(
+        "Require-Bundle", "com.google.guava", "bundle-version=\"[20.0.0,21.0.0)\"");
+  }
+
+  private void checkVersionStringsInManifestMf(
+      String attributeName, String prefixToCheck, String versionString) throws IOException {
+    String value = getManifestAttributes().getValue(attributeName);
+    if (value != null) {
+      String regexPrefix = prefixToCheck.replaceAll("\\.", "\\\\.");
+      Pattern pattern = Pattern.compile(regexPrefix + "[^;,]*");
+
+      Matcher matcher = pattern.matcher(value);
```

Comment:
Might be easier to use `Splitter.on(',')` and then apply the pattern to each import directive?

Reply:
I would have gone that way if it worked. Commas can be in, e.g., `version="[20.0.0,21.0.0)"`.

Reply:
Ah good point :-)